### PR TITLE
[RELEASE] cusignal v0.14

### PR DESCRIPTION
**Pull request by GPUtester**
### What does this mean?
Only critical/hotfix level issues should be merged into `branch-0.14` until release (merging of this PR).

### What is the purpose of this PR?
- Update documentation
- Allow testing for the new release
- Enable a means to merge `branch-0.14` into `master` for the release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,52 +118,76 @@ gf = cusignal.resample_poly(cp.asarray(cy), resample_up, resample_down, window=(
...
-## Dependencies
-* NVIDIA GPU (Maxwell or Newer GeForce/Tesla/Quadro)
-* CUDA Divers
-* Anaconda/Miniconda (3.7 version)
-* CuPy >= 6.2.0
-* Optional: RTL-SDR or other SDR Driver/Packaging. Find more information and follow the instructions for setup [here](https://github.com/osmocom/rtl-sdr). 
+## Documentation
+The complete cuSignal API documentation including a complete list of functionality and examples can be found for both the Stable and Nightly (Experimental) releases.
+
+[cuSignal 0.13 API](https://docs.rapids.ai/api/cusignal/stable/) | [cuSignal 0.14 Nightly](https://docs.rapids.ai/api/cusignal/nightly/)
```
Comment (mnicely) [inline README.md:124]: Should this be cuSignal 0.14 API and cuSignal 0.15 Nightly?

File: README.md [hunk 2]
```diff
@@ -94,52 +118,76 @@ gf = cusignal.resample_poly(cp.asarray(cy), resample_up, resample_down, window=(
...
+### Conda, Linux OS
+cuSignal can be installed with conda ([Miniconda](https://docs.conda.io/en/latest/miniconda.html), or the full [Anaconda distribution](https://www.anaconda.com/distribution/)) from the `rapidsai` channel. If you're using a Jetson GPU, please follow the build instructions [below](https://github.com/rapidsai/cusignal#conda---jetson-nano-tk1-tx2-xavier-linux-os)
+
+For `cusignal version == 0.13`:
 
-NOTE: [pyrtlsdr](https://github.com/roger-/pyrtlsdr) is not automatically installed with the default cusignal environment. To make use of some of the examples in the Notebooks, you'll need to buy/install an rtl-sdr and necessary software packages.
+```
+# For CUDA 10.0
+conda install -c rapidsai -c nvidia -c conda-forge \
+    -c defaults cusignal=0.13 python=3.6 cudatoolkit=10.0
```
Comment (mnicely) [inline README.md:136]: Again, should this be cusignal=0.14?

File: README.md [hunk 3]
```diff
@@ -94,52 +118,76 @@ gf = cusignal.resample_poly(cp.asarray(cy), resample_up, resample_down, window=(
...
+
+# or, for CUDA 10.1.2
+conda install -c rapidsai -c nvidia -c numba -c conda-forge \
+    cusignal=0.13 python=3.6 cudatoolkit=10.1
```
Comment (mnicely) [inline README.md:140]: cusignal=0.14?

File: README.md [hunk 4]
```diff
@@ -94,52 +118,76 @@ gf = cusignal.resample_poly(cp.asarray(cy), resample_up, resample_down, window=(
...
+
+# or, for CUDA 10.2
+conda install -c rapidsai -c nvidia -c numba -c conda-forge \
+    cusignal=0.13 python=3.6 cudatoolkit=10.2
```
Comment (mnicely) [inline README.md:144]: cusignal=0.14?

File: README.md [hunk 5]
```diff
@@ -94,52 +118,76 @@ gf = cusignal.resample_poly(cp.asarray(cy), resample_up, resample_down, window=(
...
-## Install cuSignal, Linux OS
+### Conda - Jetson Nano, TK1, TX2, Xavier, Linux OS
+
+While there are many versions of Anaconda for AArch64 platforms, cuSignal has been tested and supports [conda4aarch64](https://github.com/jjhelmus/conda4aarch64/releases). Conda4aarch64 is also described in the [Numba aarch64 installation instructions](http://numba.pydata.org/numba-doc/latest/user/installing.html#installing-on-linux-armv8-aarch64-platforms). Further, it's assumed that your Jetson device is running a current edition of [JetPack](https://developer.nvidia.com/embedded/jetpack) and contains the CUDA Toolkit.
 
 1. Clone the repository
 
     ```bash
-    # Set the location to cuSignal in an environment variable CUSIGNAL_HOME
+    # Set the localtion to cuSignal in an environment variable CUSIGNAL_HOME
```
Comment (mnicely) [inline README.md:174]: localtion --> location?

File: README.md [hunk 6]
```diff
@@ -261,9 +316,48 @@ In the cuSignal top level directory:
     pytest
     ```
 
+### Docker - All RAPIDS Libraries, including cuSignal
+
+For `cusignal version == 0.13`:
```
Comment (mnicely) [inline README.md:321]: 0.13 --> 0.14

**Comment by awthomp** [comment]
I concur with all of Matt's suggestions. @BradReesWork, how do we go about updating the README w.r.t. this Release PR?

**Comment by raydouglass** [comment]
> how do we go about updating the README w.r.t. this Release PR?

Open a PR against `branch-0.14` and notify me for approval to merge it.

**Comment by awthomp** [comment]
@raydouglass -- Pushed README update here: https://github.com/rapidsai/cusignal/pull/110

**Pull request merged**